packages/core/src/js/feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Image, Text, TouchableOpacity } from 'react-native';
+
+import { defaultButtonConfiguration } from './defaults';
+import { defaultButtonStyles } from './FeedbackWidget.styles';
+import type { FeedbackButtonProps, FeedbackButtonStyles, FeedbackButtonTextConfiguration } from './FeedbackWidget.types';
+import { feedbackIcon } from './icons';
+import { lazyLoadFeedbackIntegration } from './lazy';
+
+const showFeedbackWidget = (): void => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { showFeedbackWidget } = require('./FeedbackWidgetManager');
```

Comment:
The cyclic dependency is properly fixed with a small refactoring in a followup PR https://github.com/getsentry/sentry-react-native/pull/4718